packages/autocomplete-core/src/getPropGetters.ts

```diff
@@ -104,8 +109,15 @@ export function getPropGetters<
       role: 'combobox',
       'aria-expanded': store.getState().isOpen,
       'aria-haspopup': 'listbox',
-      'aria-owns': store.getState().isOpen ? `${props.id}-list` : undefined,
-      'aria-labelledby': `${props.id}-label`,
+      'aria-owns': store.getState().isOpen
+        ? store
+            .getState()
+            .collections.map(({ source }) =>
+              getAutocompleteElementId(props.id, 'list', source)
+            )
+            .join(' ')
+        : undefined,
```

Comment:
This is [conform to the spec](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-owns#values), making Autocomplete the owner of all the sources' lists.